### PR TITLE
docs: add a tenant example to prometheus.remote_write.md

### PR DESCRIPTION
**Pull request by thecoldwine**
#### PR Description

This PR adds an example for pushing the metrics to a multi-tenant Mimir instance with non-default tenant.

It could be useful for people who have less experience with multi-tenant Mimir, but would like to have a clear example how to use recommended way for ingestion there.

#### Notes to the Reviewer

There are no functional changes, just one particular example in the `prometheus.remote_write` config.

#### PR Checklist

- [x] Documentation added

### CODE REVIEW

File: docs/sources/flow/reference/components/prometheus.remote_write.md [hunk 1]
```diff
@@ -361,6 +361,25 @@ prometheus.scrape "demo" {
 }
 ```
 
+
+### Send metrics to a Mimir instance with a tenant specified
+
+You can create a `prometheus.remote_write` component that sends your metrics to a specific tenant within Mimir instance. It is useful when
```
Comment (clayton-cornell) [inline docs/sources/flow/reference/components/prometheus.remote_write.md:367]:
```suggestion
You can create a `prometheus.remote_write` component that sends your metrics to a specific tenant within the Mimir instance. It is useful when
```

**Comment by clayton-cornell** [comment]
@grafana/grafana-agent-maintainers Can one of you take a quick look over the example to double check the technical accuracy

**Review by spartan0x117 — COMMENTED** [review]
From a technical perspective looks good, just one comment around having a more generic url and comment :smiley:

File: docs/sources/flow/reference/components/prometheus.remote_write.md [hunk 2]
```diff
@@ -362,6 +362,25 @@ prometheus.scrape "demo" {
...
+your Mimir instance is using more than one tenant:
+
+```river
+prometheus.remote_write "staging" {
+  // Send metrics to a Mimir running within your kubernetes cluster, as an example.
+  endpoint {
+    url = "http://mimir-gateway.mimir.svc.cluster.local/api/v1/push"
```
[the remark this answers is not in the document]
Comment (thecoldwine) [reply docs/sources/flow/reference/components/prometheus.remote_write.md]: Good point, changed to a generic Mimir url and a less specific comment

File: docs/sources/flow/reference/components/prometheus.remote_write.md [hunk 3]
```diff
@@ -362,6 +362,24 @@ prometheus.scrape "demo" {
 }
 ```
 
+
+### Send metrics to a Mimir instance with a tenant specified
+
+You can create a `prometheus.remote_write` component that sends your metrics to a specific tenant within the Mimir instance. It is useful when your Mimir instance is using more than one tenant:
```
Comment (spartan0x117) [inline docs/sources/flow/reference/components/prometheus.remote_write.md:368]:
```suggestion
You can create a `prometheus.remote_write` component that sends your metrics to a specific tenant within the Mimir instance. This is useful when your Mimir instance is using more than one tenant:
```

**Review by spartan0x117 — APPROVED** [review]
Just a minor nit but LGTM, thank you!

**Review by clayton-cornell — APPROVED** [review]

**Comment by clayton-cornell** [comment]
@thecoldwine Thanks for your input here. Getting this PR merged into Main :-)

**Pull request merged**